src/tools/perf/perftest.c

```diff
@@ -42,13 +42,13 @@ test_type_t tests[] = {
      "get latency / bandwidth / message rate", "latency", 1},
 
     {"fadd", UCX_PERF_API_UCT, UCX_PERF_CMD_FADD, UCX_PERF_TEST_TYPE_STREAM_UNI,
-     "atomic fetch-and-add latency / rate", "latency", 1},
+     "atomic fetch-and-add latency / rate", "latency", 32},
```

Comment:
i'm not sure we should do it. maybe just print the window size in the output?
we still need a test for atomic latency..